integrations.yml

```diff
@@ -58,3 +58,9 @@
   website: "https://eazybi.com/integrations/basecamp"
   icon: "eazybi.png"
   category: 3
+- 
+  name: "AssessTEAM"
+  description: "Continuous feedback based employee evaluation system & project profitability reporting suite"
+  website: "https://www.assessteam.com"
+  icon: "Assessteam-icon256.png"
```

Comment:
Hi Jeremy,

I have added icon in the pull request now.

https://github.com/ps97-ketangohil/bc3-integrations/pull/2/commits/a30c371b04fbf1eb6f229672e36f70daee7eb7c7

Please review.

Thanks
Ketan